File: examples-sdk/csharp/702-mq-sqs-send-receive/Main.cs
// 702-mq-sqs-send-receive — C# SDK equivalent
// Compile: vil compile --from csharp --input 702-mq-sqs-send-receive/Main.cs --release

#r "sdk/csharp/Vil.cs"

var server = new VilServer("app", 8080);
server.Compile();

File: examples-sdk/java/702-mq-sqs-send-receive/Main.java
// 702-mq-sqs-send-receive — Java SDK equivalent
// Compile: vil compile --from java --input 702-mq-sqs-send-receive/Main.java --release

import dev.vil.*;

public class Main {
    public static void main(String[] args) {
        VilServer server = new VilServer("app", 8080);
        server.compile();
    }
}
